### PR TITLE
Added Makefile targets for diffing previous and current encrypted .env files

### PR DESCRIPTION
**Pull request by jimleroyer**
## What are you changing?

Added Makefile targets for diffing previous and current encrypted .env files:

### `make diff-staging`

```bash
> make diff-staging   
26c26
< FF_BATCH_INSERTION=<hidden>
---
> FF_BATCH_INSERTION=<hidden>
41a42,43
> RECAPTCHA_PUBLIC_KEY=<hidden>
> RECAPTCHA_PRIVATE_KEY=<hidden>
< ZENDESK_API_URL=<hidden>
< ZENDESK_SELL_API_KEY=<hidden>
< ZENDESK_SELL_API_URL=<hidden>
< MIXPANEL_PROJECT_TOKEN=<hidden>
< RECAPTCHA_PUBLIC_KEY=<hidden>
< RECAPTCHA_PRIVATE_KEY=<hidden>
---
> ZENDESK_API_URL=<hidden>
> ZENDESK_SELL_API_URL=<hidden>
> ZENDESK_SELL_API_KEY=<hidden>
make: *** [diff-staging] Error 1
```

### `make diff-production`

```
make diff-production                
63,64d62
< SENSITIVE_SERVICES=<hidden>
< 
73c71
< ZENDESK_API_URL=<hidden>
---
> ZENDESK_API_URL=<hidden>
< MIXPANEL_PROJECT_TOKEN=<hidden>
make: *** [diff-production] Error 1
```

## Provide some background on the changes

We got mixed up on present and missing variables in the encrypted files. This should help us diagnose and prevent in the future.

**Next steps could consist of hooking up in the workflow and prevent a merge if there is a removal of environment variable and require a force merge. /brainstorm**

## If you are releasing a new version of Notify, what components are you updating
- [ ] API
- [ ] Admin
- [ ] Documentation
- [ ] Document download API
- [ ] Document download frontend

## Checklist if releasing new version:
- [ ] I made sure that the changes are as expected in [Notify staging](https://staging.notification.cdssandbox.xyz/)
- [ ] I have checked if the docker images I am referencing exist
    - [ ] [api lambda](https://ca-central-1.console.aws.amazon.com/ecr/repositories/private/296255494825/notify/api-lambda?region=ca-central-1) (requires Notification-Production / AdministratorAccess login)
    - [ ] [api k8s](https://gallery.ecr.aws/v6b8u5o6/notify-api)
    - [ ] [admin](https://gallery.ecr.aws/v6b8u5o6/notify-admin)
    - [ ] [documentation](https://gallery.ecr.aws/v6b8u5o6/notify-documentation)
    - [ ] [document download API](https://gallery.ecr.aws/v6b8u5o6/notify-document-download-api)
    - [ ] [document download frontend](https://gallery.ecr.aws/v6b8u5o6/notify-document-download-frontend)

## Checklist if making changes to Kubernetes:
- [ ] I know how to get kubectl credentials in case it catches on fire

## After merging this PR
- [ ] I have verified that the tests / deployment actions succeeded
- [ ] I have verified that any affected pods were restarted successfully
- [ ] I have verified that I can still log into [Notify production](https://notification.canada.ca)
- [ ] I have verified that the smoke tests still pass on production
- [ ] I have communicated the release in the #notify Slack channel.

### CODE REVIEW

**Review by jzbahrai — APPROVED** [review]

**Review by iokpala — APPROVED** [review]
LGTM..Cheers Jimmy

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .DS_Store
 .env
+.previous.env
```
Comment (sastels) [inline .gitignore:3]: 😅 this is good

**Review by sastels — APPROVED** [review]
tested locally, works as advertised! just needed to add `AWS_PROFILE` before the make command.

**Review by mohamed-cds — APPROVED** [review]
This is really good @jimleroyer. Awesome work!

**Pull request merged**